py/selenium/webdriver/remote/webelement.py

```diff
@@ -450,6 +452,9 @@ def screenshot(self, filename):
             return False
         finally:
             del png
+        if not filename.lower().endswith('.png'):
```

Comment:
Can you move this warning to be the first thing that happens in the method so people can see the warning _and_ get any exceptions that might happen.